twofactor/twofactor.go

```diff
@@ -336,17 +348,29 @@ func (tf *Twofactor) Auth(electionID []byte, authToken *uuid.UUID, authData []st
 		log.Warnf("verify challenge %s failed: %v", solution, err)
 		return AuthResponse{Error: "challenge not completed:" + err.Error()}
 	}
+	r, err := tf.NewBlindRequestKey()
+	if err != nil {
+		return AuthResponse{Error: "error getting new blind token:" + err.Error()}
+	}
+	tokenR := r.BytesUncompressed()
```

Comment:
I think that the `NewBlindRequestKey` method could return the `BytesUncompressed` result instead a `blind.Point`. The `blind.Point` is never used so we can assume that our token is already the uncompressed []bytes.